### PR TITLE
Finops update 09052024

### PR DESCRIPTION
**Pull request by bandersmsft**
## 🛠️ Description

Fixes # 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [ ] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

**Comment by bandersmsft** [comment]
@flanakin This PR syncs the MS Learn content to the finops toolkit repo.

File: docs-mslearn/framework/manage/tools-services.md [hunk 1]
```diff
@@ -35,18 +35,18 @@ To clarify terminology, a FinOps "tool" is a software solution that facilitates
 
 ## Getting started
 
-When you first start managing cost in the cloud, you use the native tools available in the portal. This includes, but is not limited to:
+When you first start managing cost in the cloud, you use the native tools available in the portal, including but no limited to:
```
Comment (flanakin) [inline docs-mslearn/framework/manage/tools-services.md:38]:
```suggestion
When you first start managing cost in the cloud, you use the native tools available in the portal, including but not limited to:
```

File: docs-mslearn/framework/manage/tools-services.md [hunk 2]
```diff
@@ -65,11 +65,11 @@ Once you have a consolidated list of the Microsoft, third-party, and homegrown t
 
 ## Building on the basics
 
-At this point, you've defined your organizational objectives, identified how current tools and services meet them, and hopefully identified any limitations and gaps. As you move beyond the basics, you'll focus on establishing a plan to address limitations and gaps or opportunities to go beyond your basic requirements to further maximize cloud ROI through new opportunities made available via new or existing tools and services.
+At this point, you defined your organizational objectives, identified how current tools and services meet them, and hopefully identified any limitations and gaps. As you move beyond the basics, you focus on establishing a plan to address limitations and gaps or opportunities to go beyond your basic requirements to further maximize cloud ROI through new opportunities made available via new or existing tools and services.
 
-- Evaluate limitations and gaps in existing tools and services and establish a plan to address them.
+- Evaluate limitations and gaps in existing tools and services and establish a plan to address.
```
Comment (flanakin) [inline docs-mslearn/framework/manage/tools-services.md:70]:
```suggestion
- Evaluate existing tools and services and establish a plan to address any limitations and gaps.
```

File: docs-mslearn/framework/optimize/sustainability.md [hunk 3]
```diff
@@ -54,7 +54,6 @@ This capability is a part of the FinOps Framework by the FinOps Foundation, a no
 
 Related FinOps capabilities:
 
-- [Managing commitment discounts](./commitment-discounts.md)
 - [Workload management and automation](./workloads.md)
```
Comment (flanakin) [inline docs-mslearn/framework/optimize/sustainability.md:57]:
```suggestion
- [Workload optimization](./workloads.md)
- [Rate optimization](./rates.md)
```

File: docs-mslearn/framework/understand/ingestion.md [hunk 4]
```diff
@@ -28,9 +28,9 @@ Gather cost, utilization, performance, and other business data from cloud provid
 
 Understand how data gets reported and plan for data standardization requirements to support reporting on similar data from multiple sources. 
 
-Consider how to handle cost data from multiple clouds or account types. Prefer open standards, like the FinOps Open Cost & Usage Specification ([FOCUS project](../../../focus/README.md)), which delivers consistency and standardization to cloud cost data, and interoperability with and across providers, vendors, and internal tools.
+Consider how to handle cost data from multiple clouds or account types. Prefer open standards, like the FinOps Open Cost & Usage Specification [FOCUS project](https://azure.microsoft.com/updates/export-cost-using-focus/), which delivers consistency and standardization to cloud cost data, and interoperability with and across providers, vendors, and internal tools.
```
Comment (flanakin) [inline docs-mslearn/framework/understand/ingestion.md:31]:
```suggestion
Consider how to handle cost data from multiple clouds or account types. Prefer open standards, like the [FinOps Open Cost & Usage Specification (FOCUS) project](https://aka.ms/ftk/focus), which delivers consistency and standardization to cloud cost data, and interoperability with and across providers, vendors, and internal tools.
```

File: docs-mslearn/framework/understand/ingestion.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ When armed with a comprehensive collection of cost and usage information tied to
...
 
-During the first iteration to start adopting this capability, consider using FOCUS as the standard billing data format for all of your data sources. To learn why organizations needs it and why Microsoft believes in FOCUS, you can review the [FOCUS documentation](../../../focus/README.md) available on FinOps toolkit provided by Microsoft.
+During the first iteration to start adopting this capability, consider using FOCUS as the standard billing data format for all of your data sources. To learn why organizations need it and why Microsoft believes in FOCUS, you can review the [FOCUS documentation](https://azure.microsoft.com/updates/export-cost-using-focus/) available on FinOps toolkit provided by Microsoft.
```
Comment (flanakin) [inline docs-mslearn/framework/understand/ingestion.md:43]:
```suggestion
During the first iteration to start adopting this capability, consider using FOCUS as the standard billing data format for all of your data sources. To learn why organizations need it and why Microsoft believes in FOCUS, you can review the [FOCUS documentation](https://aka.ms/ftk/focus) available on FinOps toolkit provided by Microsoft.
```

File: docs-mslearn/framework/understand/reporting.md [hunk 6]
```diff
@@ -54,51 +53,55 @@ Before you can effectively analyze usage and costs, you need to familiarize your
...
   - "Amortized" cost shows the effective cost of resources that used a commitment-based discount (reservation or savings plan). Use amortized costs for cost allocation, to "smooth out" large purchases that might look like usage spikes, and numerous commitment-based discount scenarios.
 - How credits are applied. For instance, credits are applied when the invoice is generated and not when usage is tracked.
 
 Understanding your cost data is critical to enable accurate and meaningful showback to all stakeholders.
 
-If there is the need to report from multiple sources, for example multiple cloud providers, configuration management database (CMDB), or  IT asset management (ITAM) systems, consider using the FinOps Open Cost & Usage Specification ([FOCUS project](../../../focus/README.md)) as the standard billing schema for this report.
+Consider using the FinOps Open Cost & Usage Specification [FOCUS project](https://azure.microsoft.com/updates/export-cost-using-focus/) as the standard billing schema for your report if you need to report from multiple sources, such as:
```
Comment (flanakin) [inline docs-mslearn/framework/understand/reporting.md:68]:
```suggestion
Consider using the [FinOps Open Cost & Usage Specification (FOCUS) project](https://aka.ms/ftk/focus) as the standard billing schema for your report if you need to report from multiple sources, such as:
```

**Review by flanakin — APPROVED** [review]

**Pull request merged**